src/main/scripts/festerize/README.md

```diff
@@ -4,11 +4,11 @@ Uploads CSV files to the Fester IIIF manifest service for processing.
 
 Any collection rows found in the CSV are used to create a IIIF collection.
 
-Any work rows are used to expand or revise a previously created IIIF collection (corresponding to the collection that the work is a part of), as well as create a IIIF manifest corresponding to the work.
+Any work rows are used to expand or revise a previously created IIIF collection (corresponding to the collection that the work is a part of), as well as create a IIIF manifest corresponding to the work. A "work" is conceived of as a discrete object that should be access as an individual item. An example of a "work" is a book or a photograph. 
 
-Any page rows are likewise used to expand or revise a previously created IIIF manifest (corresponding to the work that the page is a part of).
+Any page rows found in a CSV are likewise used to expand or revise a previously created IIIF manifest (corresponding to the work that the page is a part of).
```

Comment:
RE: your comment in Jira @emcaulay -- yes, this wording is incorrect. Currently, if you give Fester a CSV with both work rows and page rows, it will ignore the page rows. If you want it to do something with the page rows, it must not contain any work rows (or collection rows, for that matter). I will fix this to correctly describe the current behavior.

Reply:
If you give Fester a CSV with Collection, Work, and Page rows, it should process all.

Reply:
Okay, I must be reading the code wrong then. I think you're right @ksclarke and I think my first comment is incorrect.